middle_end/flambda2/from_lambda/lambda_to_flambda.ml

```diff
@@ -1017,6 +1025,14 @@ let name_if_not_var acc ccenv name simple kind body =
       [id, kind]
       Not_user_visible (IR.Simple simple) ~body:(body id)
 
+let is_user_visible env id ~(default : IR.user_visible) : IR.user_visible =
```

Comment:
What's the point of the `default` parameter ? It's always passed as `User_visible` currently.

Reply:
Ah, indeed - that can be removed now then.